### DISCUSSION TITLE
How to import indicator data collected from Skywalking into Prometheus

### DISCUSSION THREAD

**Discussion by liuhao710118** [apache/skywalking#12536] [Q&A] [answered]
Skywalking version 9.7

My company needs to do application monitoring, which I easily achieved through Skywalk. Now I need to calculate global indicators, and I need to import the indicators calculated by Skywalk into Prometheus. I found that there was a Prometheus fetcher before version 9.3, which was removed after 9.3. I recommend using Open Telemetry to implement it, but I am not very familiar with Open Telemetry, so I am here to seek help from all the experts. Is there anyone who has done a similar task before? I need help

**Comment by wu-sheng** [comment]
Your asking doesn't make sense to me.

The fetcher we removed is used to read Prometheus endpoint metrics into skywalking, rather than reverted.

I don't know what is global indicator, we don't have that concept.

**Reply by liuhao710118** [reply]
My fundamental intention is to export metrics from Skywalking to Prometheus for analysis

**Reply by wu-sheng** [reply, marked as answer]
Write your exporter adaptor to any place or format you need.

https://skywalking.apache.org/docs/main/latest/en/setup/backend/exporter/

**Comment by lpq131004** [comment]
@liuhao710118 Have you solved this problem?

**Reply by liuhao710118** [reply]
I finally obtained the data by directly checking the Elasticsearch database
